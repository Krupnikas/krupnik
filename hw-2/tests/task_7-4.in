7HeLlO,_WoRlD!8
