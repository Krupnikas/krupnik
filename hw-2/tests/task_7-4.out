7SeOlL,_DoIlW!8
